packages/amplify-dotnet-function-template-provider/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "amplify-dotnet-function-template-provider",
-  "version": "1.0.1",
+  "version": "1.0.2",
```

Comment:
lerna manages the package versions for us, so leave this unchanged

Reply:
Oop! Reverting...